inception-project/inception
# #905 - Offset for recommendation rendering

**What's in the PR**
* adapt changes from webanno/webanno#1212
* fix bug where span annotation texts contained ` and were not escaped

**How to test manually**
* Upstream changes from webanno/webanno#1212 required
* Open a PDF in PDF editor
* Trigger rendering for recommendations (e.g. by opening sidebar)
* See that the whole document is filled with recommendations

**Automatic testing**

**Documentation**
